### DISCUSSION TITLE
CPU fast (PL2) sliders

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#1894] [General]
![image](https://github.com/seerge/g-helper/assets/114208757/5ab1ca33-e834-49e3-93ca-becab1ba9f4c)

pl2 setting is added but has no effect.

i5 10300h CPU - ASUS TUF 506LI

**Comment by seerge** [comment]
@hadixcanimm hello, it shouldn't be visible for you (as your device doesn't support that), sorry :) 

I have updated 0.146 with a fix
https://github.com/seerge/g-helper/releases/download/v0.146/GHelper.zip

Thanks for spotting
